src/faldbt/project.py

```diff
@@ -358,6 +359,23 @@ def _find_features(self) -> List[Feature]:
                 )
         return features
 
+    def _model(
```

Comment:
should we call these `_get_model` and the other one `_get_source` non very method looks weird

Reply:
we can do that for sure. I have a refactor PR coming later, we can do it here